utils/helpers.js

```diff
@@ -89,3 +89,8 @@ export const truncateDescription = (desc = '', maxLength, isOpen) => {
   const ellipsis = isOpen ? '' : '...'
   return { truncated: desc.slice(0, lastSpaceIndex) + ellipsis, cutOffIndex: lastSpaceIndex }
 }
+
+export const removeHtmlSpaces = (text) => {
+  const regex = /&[a-z]+;/g
```

Comment:
I suspect this regex is going to cause unexpected bugs because the & may be in a ware description. If you are only stripping `&nbsp` why don't you just match on that?

Reply:
another solution perhaps is that you can find a way to read html entities?

Reply:
this will only apply to the snippet, not the description. this particular regex would catch other html entities that we may not want, but not remove all instances of an ampersand. only cases where an ampersand is followed by letters and ends with a semicolon.
![image](https://github.com/scientist-softserv/phenovista-digital-storefront/assets/29032869/2c30bdde-1379-41e1-99c1-125ea4f947cf)

Reply:
true, reading the entities is also an option. I felt this one was quicker. but can visit the other idea if time permits.

Reply:
maybe https://github.com/mathiasbynens/he would work?

Regex will come back to haunt you later (says someone who may have been haunted by regex)